### PR TITLE
3080 modeler to console

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-docs/issues/3080.

@HanselIdes once approved I will backport 👍 

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by HanselIdes — APPROVED** [review]
@christinaausley LGTM! There's an upcoming change (https://github.com/camunda/product-hub/issues/1973) that will change the cluster creation flow, but it can be addressed later

**Review by akeller — APPROVED** [review]
Looks like this is good to go, just needs to be backported.

File: src/pages/contact.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ There are a few different channels you can reach us based on your needs:
 
 - For **security-related issues**, review our [security notices](../docs/reference/notices) for the most up-to-date information on known issues and steps to report a vulnerability so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
 
-- **Feedback and support** can be submitted or requested via JIRA for subscription or enterprise customers in the Support project. Otherwise, use the [Camunda community forum](https://forum.camunda.io/). For more information about Enterprise support and additional support resources, see [Enterprise Support](https://camunda.com/support/).
+- **Feedback and support** can be submitted or requested via JIRA for subscription or enterprise customers in the Support project. All users can also find feedback and support options in the [Help Center](https://camunda.com/services/support/) or [Camunda community forum](https://forum.camunda.io/).
```
Comment (akeller) [inline src/pages/contact.md:16]: Help Center is an area in the SaaS product, so no link is necessary here.

```suggestion
- **Feedback and support** can be submitted or requested via JIRA for subscription or enterprise customers in the Support project. All users can also find feedback and support options in the Help Center or [Camunda community forum](https://forum.camunda.io/).
```

**Comment by christinaausley** [comment]
@HanselIdes @akeller Mind giving me one more thumbs up here?

**Review by akeller — APPROVED** [review]

**Pull request merged**